lib/stripe.js

```diff
@@ -49,6 +50,9 @@ const {emitWarning} = utils;
 Stripe.StripeResource = require('./StripeResource');
 Stripe.resources = resources;
 
+Stripe.HttpClient = require('./net/HttpClient');
+Stripe.NodeHttpClient = require('./net/NodeHttpClient');
```

Comment:
This will fail in non-Node environments. Maybe instead of exposing an instantiated `NodeHttpClient` we could expose a `createNodeHttpClient` function?

```js
Stripe.createNodeHttpClient = (agent) => {
   const NodeHttpClient = require('./net/NodeHttpClient');
   return new NodeHttpClient(agent);
};
```

Reply:
Done. Good call.